src/three/PNTSLoader.js

```diff
@@ -1,53 +1,84 @@
 import { PNTSLoaderBase } from '../base/PNTSLoaderBase.js';
-import { Points, PointsMaterial, BufferGeometry, BufferAttribute, DefaultLoadingManager } from 'three';
-
+import {
+	Points,
+	PointsMaterial,
+	BufferGeometry,
+	BufferAttribute,
+	DefaultLoadingManager,
+} from 'three';
+
+const DRACO_ATTRIBUTE_MAP = {
+	RGB: 'color',
+	POSITION: 'position',
+};
 export class PNTSLoader extends PNTSLoaderBase {
 
 	constructor( manager = DefaultLoadingManager ) {
 
 		super();
 		this.manager = manager;
 
+		// very hacky way of getting the draco loader
+		this.dracoLoader = this.manager.getHandler( 'test.drc' );
+
```

Comment:
Should we use something more descriptive than `test.drc`? Maybe `draco.drc`?

Reply:
sure